install/running_install.adoc

```diff
@@ -165,41 +137,35 @@ ifdef::openshift-enterprise[]
     /usr/share/ansible/openshift-ansible/playbooks/prerequisites.yml
 endif::[]
 ifdef::openshift-origin[]
-# ansible-playbook [-i /path/to/inventory] \
+# ansible-playbook [-i /path/to/inventory] \ <1>
     ~/openshift-ansible/playbooks/prerequisites.yml
 endif::[]
 ----
-<1>  If your inventory file is not in the *_/etc/ansible/hosts_* directory, 
+<1> If your inventory file is not in the *_/etc/ansible/hosts_* directory, 
 specify `-i` and the path to the inventory file.
 
 . Run the *_deploy_cluster.yml_* playbook to initiate the cluster installation:
 +
 ----
 ifdef::openshift-enterprise[]
-# ansible-playbook [-i /path/to/inventory] \
+# ansible-playbook [-i /path/to/inventory] \ <1>
     /usr/share/ansible/openshift-ansible/playbooks/deploy_cluster.yml
 endif::[]
 ifdef::openshift-origin[]
-# ansible-playbook [-i /path/to/inventory] \
+# ansible-playbook [-i /path/to/inventory] \ <1>
     ~/openshift-ansible/playbooks/deploy_cluster.yml
 endif::[]
 ----
+<1> If your inventory file is not in the *_/etc/ansible/hosts_* directory, 
+specify `-i` and the path to the inventory file.
 
-If for any reason the installation fails, before re-running the installer, see
-xref:installer-known-issues[Known Issues] to check for any specific
-instructions or workarounds.
-
-[WARNING]
-====
-The installer caches playbook configuration values for 10 minutes, by default. If you change any system, network, or inventory configuration,
-and then re-run the installer within that 10 minute period, the new values are not used, and the previous values are used instead.
-You can delete the contents of the cache, which is defined
-by the `fact_caching_connection` value in the *_/etc/ansible/ansible.cfg_* file. An example of this file is
-shown in xref:../scaling_performance/install_practices.adoc#scaling-performance-install-optimization[Recommended Installation Practices].
-====
+. 
+** If your installation succeeded,
+xref:advanced-verifying-the-installation[verify the installation].
+** If your installation failed, xref:advanced-retrying-installation[retry the installation].
 
 [[running-the-advanced-installation-containerized]]
-=== Running the Containerized Installer
```

Comment:
Is everything in this section still valid?

Reply:
@kalexand-rh LGTM